### PR TITLE
Update dependencies and related code

### PR DESCRIPTION
**Pull request by intricate**

### CODE REVIEW

File: cardano-api/src/Cardano/Api/TxSubmit.hs [hunk 1]
```diff
@@ -72,8 +73,8 @@ submitTx network socketPath tx =
                       socketPath
                       genTx
           case result of
-            Nothing  -> return TxSubmitSuccess
-            Just err -> return (TxSubmitFailureByron err)
+            SubmitSuccess  -> return TxSubmitSuccess
+            SubmitFail err -> return (TxSubmitFailureByron err)
```
Comment (erikd) [inline cardano-api/src/Cardano/Api/TxSubmit.hs:77]: :100: :100: :100:

File: cardano-api/test/Test/Cardano/Api/Convert.hs [hunk 2]
```diff
@@ -14,8 +14,10 @@ import qualified Hedgehog as Hedgehog
 
 import           Test.Cardano.Api.Gen
 
-prop_roundtrip_AddressByron_hex :: Property
-prop_roundtrip_AddressByron_hex =
+-- This test has been disabled because 'Shelley.Spec.Ledger.Address.getByron'
+-- has not been implemented yet and just 'panic's.
+_prop_roundtrip_AddressByron_hex :: Property
+_prop_roundtrip_AddressByron_hex =
```
Comment (erikd) [inline cardano-api/test/Test/Cardano/Api/Convert.hs:20]: :+1:

**Comment by intricate** [comment]
Awaiting the merge of https://github.com/input-output-hk/ouroboros-network/pull/2094 to remedy the block forging problem.

**Comment by nc6** [comment]
That PR is now merged!

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**